src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -187,7 +201,8 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
                 shouldCheckStreaming = false;
             }
             try {
-                KustoOperationResult rs = engineClient.execute(database, String.format(FETCH_TABLE_COMMAND, table), validateOnlyClientRequestProperties);
+                KustoOperationResult rs = engineClient.execute(database, String.format(FETCH_TABLE_COMMAND, table),

```

Comment:
Note: Using the `mvn formatter:format`, I still get some differences, not sure why